.github/workflows/create-release-draft.yml

```diff
@@ -186,6 +184,73 @@ jobs:
         run: |
           make clean-docker-${{ matrix.chain }}
 
+  ## check extrinsic ##
+  extrinsic-ordering-check-from-bin:
+    runs-on: self-hosted
```

Comment:
Will it work on ubuntu-latest? If so I'd prefer that, but if it needs something from our host, we can simply leave it.

Reply:
should work as well. Let me update it.